build.rb

```diff
@@ -25,18 +26,26 @@
 
 WINDOWS = %w[
   setup_instructions
+  setup/wsl2_prereq_intro
+  setup/wsl2_prereq_win10
+  setup/wsl2_prereq_win_version
+  setup/wsl2_prereq_virtualization
+  setup/github
   setup/remote_tools
-  win_sublime
-  win_github_account
-  win_git
-  win_github
-  win_python
-  win_virtualenv
+  setup/wsl2_install_wsl
+  setup/wsl2_vscode
+  setup/wsl2_windows_terminal
+  setup/wsl2_git
+  setup/wsl2_oh_my_zsh
+  github_rsa
+  dotfiles
+  setup/wsl_browser_variable
+  ubuntu_python
+  osx_virtualenv
   pip
-  win_sublime_package
-  win_make
-  win_extra
-  win_docker
+  ubuntu_docker
+  setup/ubuntu_inotify
+  setup/ubuntu_extra
```

Comment:
Windows setup does not require the codec. Students will open the video on their browser installed on Windows directly, not through WSL 👍

Reply:
Happy to oblige 👍